app/src/main/java/org/oppia/android/app/devoptions/forcenetworktype/ForceNetworkTypeFragment.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.android.app.devoptions.forcenetworktype
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment to provide functionality to force network type of the app. */
```

Comment:
```suggestion
/** Fragment to provide functionality to force the network type of the app. */
```

Small nit on grammar.

Reply:
Done.